src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,154 @@
+                new User(CORRECT_LOGIN, CORRECT_PASSWORD, BELOW_MINIMUM_AGE)));
+    }
+
+    @Test
+    void register_correctLogin_Ok() {
+        registrationService.register(new User(LOGIN_YURI, CORRECT_PASSWORD, MINIMUM_AGE));
+        registrationService.register(new User(LOGIN_DASHA, CORRECT_PASSWORD, MINIMUM_AGE));
+        registrationService.register(new User(LOGIN_NAZAR, CORRECT_PASSWORD, MINIMUM_AGE));
+        registrationService.register(new User(
+                CORRECT_LOGIN_SIX_LETTERS, CORRECT_PASSWORD, MINIMUM_AGE));
+        assertTrue(people.contains(new User(LOGIN_YURI, CORRECT_PASSWORD, MINIMUM_AGE)));
+        assertTrue(people.contains(new User(LOGIN_DASHA, CORRECT_PASSWORD, MINIMUM_AGE)));
+        assertTrue(people.contains(new User(LOGIN_NAZAR, CORRECT_PASSWORD, MINIMUM_AGE)));
+        assertTrue(people.contains(new User(
+                CORRECT_LOGIN_SIX_LETTERS, CORRECT_PASSWORD, MINIMUM_AGE)));
+        assertEquals(EXPECTED_FOUR, people.size());
+    }
+
+    @Test
+    void register_incorrectLogin_NotOk() {
+        assertThrows(RegistrationException.class, () -> registrationService.register(
+                new User(INCORRECT_LOGIN, CORRECT_PASSWORD, MINIMUM_AGE)));
+    }
+
+    @Test
+    void register_incorrectLoginEmptyLine_NotOk() {
+        assertThrows(RegistrationException.class, () -> registrationService.register(
+                new User(EMPTY_LINE, CORRECT_PASSWORD, MINIMUM_AGE)));
+    }
+
+    @Test
+    void register_correctPassword_Ok() {
+        registrationService.register(new User(LOGIN_YURI, CORRECT_PASSWORD, MINIMUM_AGE));
+        registrationService.register(new User(
+                LOGIN_DASHA, CORRECT_PASSWORD_SIX_LETTERS, MINIMUM_AGE));
+        assertTrue(people.contains(new User(LOGIN_YURI, CORRECT_PASSWORD, MINIMUM_AGE)));
+        assertTrue(people.contains(new User(
+                LOGIN_DASHA, CORRECT_PASSWORD_SIX_LETTERS, MINIMUM_AGE)));
+        assertEquals(EXPECTED_TWO, people.size());
+    }
+
+    @Test
+    void register_incorrectPassword_NotOk() {
+        assertThrows(RegistrationException.class, () -> registrationService.register(
+                new User(CORRECT_LOGIN, INCORRECT_PASSWORD, MINIMUM_AGE)));
+    }
+
+    @Test
+    void register_incorrectPasswordEmptyLine_NotOk() {
+        assertThrows(RegistrationException.class, () -> registrationService.register(
+                new User(CORRECT_LOGIN, EMPTY_LINE, MINIMUM_AGE)));
+    }
+
+    @Test
+    void register_userAlreadyExists_NotOk() {
+        people.add(new User(LOGIN_DASHA, CORRECT_PASSWORD, MINIMUM_AGE));
+        assertThrows(RegistrationException.class, () -> registrationService.register(
+                new User(LOGIN_DASHA, CORRECT_PASSWORD, MINIMUM_AGE)));
+    }
+
+    @Test
+    void register_addCorrectUser_Ok() {
+        User actual = registrationService.register(new User(LOGIN_YURI,CORRECT_PASSWORD_SIX_LETTERS,MINIMUM_AGE));
```

Comment:
This test is named `register_addCorrectUser_Ok` which is good, but it would be better to specify what aspect of the user is 'correct' for clarity, such as `register_userWithValidData_Ok`.